test/components/availability_component_test.rb

```diff
@@ -26,7 +26,7 @@ class AvailabilityComponentTest < ViewComponent::TestCase
     render_inline(AvailabilityComponent.new(developer: @developer))
 
     assert_no_selector("svg")
-    assert_selector("span", text: /Available\s*in\s*2 months/m)
+    assert_selector("span", text: /Available\s*in(\sabout){0,1}\s*2 months/m)
```

Comment:
```suggestion
    assert_selector("span", text: /Available\s*in about\s*2 months/m)
```

This passed 20x in a row for me. Does it work for you?

Reply:
@joemasilotti I was going to call this out in a separate issue actually. Yes, your version should work too with how the availability is currently coming back. Problem is, I have no idea why the span suddenly changed from ‘in two months’ to ‘in about two months’ - I didn’t make any changes to my code that would have caused it, which makes me think it might be due to an incorrectly set up fixture that might have its availability output change again in the future (ie change to ‘in 1 month.’)  

The other possibility is that February being an unusually short month changes the logic of Timecomponent’s output. If the content of the span changes based on which months of the year are ahead at the time the test is run, you’re probably going to want to keep my regex in there or change the availability date so that the output always has ‘about’ in it. (But I haven’t had time to troubleshoot as I’m try to stay focused on the initial PR review issues.)

Reply:
OK @joemasilotti I just looked more closely at the test (the answer I needed was just a couple lines above, duh!) and have found some unexpected behavior within the test suite.

@developer.available_on = Date.today + 2.months : 'Available in about 2 months'
@developer.available_on = Date.today + 45.days (up to 60 days) : 'Available in about 2 months'
@developer.available_on = Date.today + 61.days (up to 75 days): 'Available in 2 months'

What is happening is that + 2.months handles February weirdly. Today is 12/31, adding 2 months would bring me to 2/31.... since 2/31 doesn't exist, what is happening is that it is just defaulting to the last day in February. However, the time_in_words is being generated based on the number of days between today and the available_on date.

I'd suggest changing the test to add a specific number of days to today's date, OR leaving the regex in. If you change it to absolutely test for the span 'in about 2 months', the test is going to fail again starting tomorrow.

Reply:
Ah, nice find! Let's change the date manipulation so it doesn't ever render the "about", regardless of February and such.

Reply:
Sounds good - I've updated the code locally and will include that change on an upcoming 'miscellaneous PR review fixes' commit. Marking as resolved to keep my to do list clean.